lib/pretty-print.js

```diff
@@ -10,7 +10,37 @@ const pathDelimiter       = '/'
 const pathRegExp          = /(?=\/)/
 /* eslint-enable */
 
-function prettyPrintRoutesArray (routeArray) {
+function parseFunctionName (fn) {
+  let fName = fn.name || 'anonymous'
+
+  fName = fName.replace('bound ', '')
+  fName += '()'
+  return fName
+}
+
+function buildMetaObject (route, metaArray) {
+  const out = {}
+  if (!Array.isArray(metaArray)) metaArray = route.store ? Object.keys(route.store) : []
+  metaArray.forEach(m => {
+    if (route.store && route.store[m]) {
+      const curr = route.store[m]
+      if (Array.isArray(curr)) {
+        out[m] = curr.map(meta => {
+          if (typeof meta === 'function') return parseFunctionName(meta)
+          return meta
```

Comment:
This function could be called at line 32 to avoid repetition

Reply:
Good catch, that was getting a little muddy, I moved the type-checking into its own recursive function for sanity